concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1222,15 +1238,82 @@ jobs:
               export CONCOURSE_ATC_PASSWORD
               ./paas-cf/concourse/scripts/bosh-cli.sh bosh status
 
-
-  - name: tag-release
+  - name: performance-tests
     plan:
       - aggregate:
           - get: pipeline-trigger
             passed: ['post-deploy', 'smoke-tests', 'acceptance-tests', 'custom-acceptance-tests', 'bosh-cli-test']
             trigger: true
           - get: paas-cf
             passed: ['post-deploy', 'smoke-tests', 'acceptance-tests', 'custom-acceptance-tests', 'bosh-cli-test']
+          - get: cf-manifest
+            passed: ['post-deploy', 'smoke-tests', 'acceptance-tests', 'custom-acceptance-tests']
+          - get: bosh-CA
+      - task: generate-test-config
+        config:
+          platform: linux
+          image: docker:///ruby#2.2-slim
+          inputs:
+            - name: paas-cf
+            - name: cf-manifest
+          outputs:
+            - name: test-config
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                export CF_MANIFEST=cf-manifest/cf-manifest.yml
+                ./paas-cf/tests/performance-tests/generate_test_config.rb \
+                  > test-config/config.json
+
```

Comment:
> why does the bucket name end in -state ? Looks like it's not storing any state...

We are only allowed to create buckets with a `-state` in the name :)

> why not just have a "folder" in existing state bucket?

Performance data is not a state. 

> why do we want to do this? If for possible later analysis, isn't that just unnecessary luggage to carry for now?

Yes, that was the idea. To keep it for analysis so we are able to graph historical data.